.github/workflows/dashboards-issue-add-label.yml

```
@@ -4,7 +4,6 @@ on:
     types: [opened, closed, edited, reopened, assigned, unassigned, labeled, unlabeled]
 
```

You need to add permissions to this workflow to be able to pull the secrets from vault
```suggestion
permissions:
  contents: read
  id-token: write
```